### ISSUE TITLE
need to add a data folder

### ISSUE THREAD

**Issue by 2017rainman** [S120/benchmark#2]
it seems we need to add a folder named data, otherwise it will report errors.
![data-folder-errors](https://user-images.githubusercontent.com/26185823/33643950-f3277886-da7c-11e7-979b-29004f68da00.png)

**Comment by alessandrocaiani** [comment]
Right. It's seems the "data" folder got lost while updating due to an issue of Github in storing empty folders. We now added it manually from the Github website inside the "benchamrk" main folder. You can also create it locally on your PC and it should work. Thank you for let us notice it.
